faucet/src/faucet.rs

```diff
@@ -26,7 +26,10 @@ use jf_cap::{
     structs::{AssetCode, FreezeFlag},
 };
 use rand::distributions::{Alphanumeric, DistString};
-use reef::traits::Validator;
+
+// XXX looks wrong, but compilation fails if using reef::traits::Validator instead.
+use seahorse::reef::traits::Validator;
```

Comment:
@jbearer Not sure if this makes sense. But it wouldn't compile with `reef::traits::Validator`.

Reply:
Oh. Faucet seems to be using an outdated reef version.